pkg/result/filter.go

```diff
@@ -75,12 +75,36 @@ func FilterResult(ctx context.Context, result *types.Result, ignoreConf IgnoreCo
 	sort.Sort(types.BySeverity(filteredVulns))
 
 	result.Vulnerabilities = filteredVulns
+	result.MisconfSummary = calcMisconfSummary(result, misconfSummary, filteredMisconfs)
 	result.Misconfigurations = filteredMisconfs
-	result.MisconfSummary = misconfSummary
 
 	return nil
 }
 
+func calcMisconfSummary(result *types.Result, misconfSummary *types.MisconfSummary, filteredMisconfs []types.DetectedMisconfiguration) *types.MisconfSummary {
```

Comment:
Can't we just increment exceptions here?
https://github.com/aquasecurity/trivy/blob/e40e1d6412ccc7be028f600e7818653949e898ff/pkg/result/filter.go#L163-L191

Reply:
We only know if the ignore policy applies or not to a misconfiguration once we make this call https://github.com/aquasecurity/trivy/blob/eea7d1558cf2ad81ee95bf315d580084bbea5684/pkg/result/filter.go#L273

Which is called here 
https://github.com/aquasecurity/trivy/blob/eea7d1558cf2ad81ee95bf315d580084bbea5684/pkg/result/filter.go#L70

Reply:
Oh, you're right. What if counting ignored results here?
https://github.com/aquasecurity/trivy/blob/d70b1ccd88f10730e8ca159a9a911de285a4faac/pkg/result/filter.go#L274-L276

Can't we return misconfSummary from `applyPolicy`?

Reply:
You're right I pushed some changes here https://github.com/aquasecurity/trivy/pull/5359/commits/4626583d726baf0fdada76b6a53edc48a6ab1252